README.md

```diff
@@ -6,10 +6,30 @@ This repository implements GFlowNets, generative flow networks for probabilistic
 
 ### pip
 
+Prereqs: install `fftw3`:
+
+```bash
+brew install fftw  # OSX
+sudo apt-get install fftw  # Ubuntu
+```
```

Comment:
What is `fftw`? I've never used this.

Reply:
It was a dependency for one of the packages `gflownet` imports. It's a [library for DFT](https://www.fftw.org/).

 You likely already have it installed on your machine and/or the cluster.

`dpkg -l | grep fftw`

Reply:
Interesting. I wonder what uses this... :sweat_smile: